setup.py

```diff
@@ -95,7 +95,10 @@ def run(self):
 # google-api-python-client is internal dependency of pydrive. After merge of
 # https://github.com/gsuitedevs/PyDrive/pull/180 into pydrive's master,
 # usage of google-api-python-client can be removed from DVC.
-gdrive = ["pydrive==1.3.1", "google-api-python-client>=1.2"]
+gdrive = [
+    "pydrive @ git+https://github.com/gsuitedevs/PyDrive@master#egg=pydrive",
```

Comment:
should we update conda? other packages?

Reply:
I guess, yes, will check what version of PyDrive is in conda-forge for the moment. No others packages update is needed, I think.